controllers/controllers.js

```diff
@@ -10,3 +11,11 @@ exports.getTopics = (req, res) => {
       });
 }
 
+exports.getApiInfo = (req, res, next) => {
+    console.log('in controller')  
```

Comment:
Please remove all logs from the finished code!

Reply:
That's perfect, thanks! I forgot all about the console.logs as soon as I sent them so thank you for the reminder! I'll remove them, and all in future before pushing.